src/ansys/tools/repo_sync/repo_sync.py

```diff
@@ -32,6 +33,37 @@ def _ignore_patterns(path, names):
     return _ignore_patterns
 
 
+def delete_folder_contents(folder_path):
+    """Delete the content inside a folder without deleting the folder itself.
+
+    Parameters
+    ----------
+    folder_path : str
+        Path to the folder whose content is requested to be deleted.
+
+    """
+    # Check if the folder exists
+    if not os.path.exists(folder_path):
+        return

```

Comment:
Throwing some warning here to let users know that the folder does not exist might be a good idea in case there is some typo in the path by mistake.

Reply:
Sounds good. On it.